### ISSUE TITLE
Json Editor used to build Json Schema

### ISSUE THREAD

**Issue by eduo** [json-editor/json-editor#178]
Hello.

I know this is irregular but I'm sure some time ago I saw an interesting implementation of json-editor that was used to build json-schema that could be loaded into json-editor.

I can't find this "json-schema builder" based on json-editor. Do you know if I imagined it or if it exists?

It looked like the online editor, but you could add and remove different types of fields that would in turn generate the json schema that would replicate the form afterwards.

**Comment by pmk65** [comment]
One of these?
https://niebert.github.io/JSON2Schema/
http://json.bootsphp.com/jsonSchemaCreator/

**Comment by pmk65** [comment]
Added info to Wiki.

**Comment by eduo** [comment]
Hi. Just got around to checking this out today.

I don't think this covers what I was looking for.

Essentially I was looking for a web that allowes the creation of the form itself, and the output was the final JSON Schema. I'm sure I used such a tool at some point, but I can't find it any more.

The two links above are nice in that the first one converts a JSON object into a probable JSON Schema one and the second one allows interactive creation by hand of the Schema.

What I was sure I had used was an interactive form that allowed the creation of the final form (by using a form that allowed the addition of fields, checkboxes, etc.) and get the final JSON Schema at the end.
